src/main/java/ch/njol/skript/expressions/ExprFurnaceSlot.java

```diff
@@ -156,95 +125,94 @@ public Class<? extends Slot> getReturnType() {
-						// see https://minecraft.wiki/w/Smelting#Fuel
-						// this is declared here because setting the amount to 0 may cause the ItemStack to become AIR
-						Material newMaterial = fuel.getType() == Material.LAVA_BUCKET ? Material.BUCKET : Material.AIR;
-						fuel.setAmount(fuel.getAmount() - 1);
-						if (fuel.getAmount() == 0)
-							fuel = new ItemStack(newMaterial);
-						return fuel;
-					}
-					return super.getItem();
-				case RESULT:
-					if (event instanceof FurnaceSmeltEvent) {
-						ItemStack result = ((FurnaceSmeltEvent) event).getResult().clone();
-						if (isResultSlot) { // Special handling for getting the result slot
-							ItemStack currentResult = ((FurnaceInventory) getInventory()).getResult();
-							if (currentResult != null)
-								currentResult = currentResult.clone();
-							if (getTime() != EventValues.TIME_FUTURE) { // 'past result slot' and 'result slot'
-								return currentResult;
-							} else if (currentResult != null && currentResult.isSimilar(result)) { // 'future result slot'
-								currentResult.setAmount(currentResult.getAmount() + result.getAmount());
-								return currentResult;
-							} else {
-								return result;
-							}
+            return switch (slot) {
+                case ORE -> {
+                    if (event instanceof FurnaceSmeltEvent furnaceSmeltEvent) {
+                        ItemStack source = furnaceSmeltEvent.getSource().clone();
+                        if (getTime() != EventValues.TIME_FUTURE)
+                            yield source;
+                        source.setAmount(source.getAmount() - 1);
+                        yield source;
+                    }
+                    yield super.getItem();
+                }
+                case FUEL -> {
+                    if (event instanceof FurnaceBurnEvent furnaceBurnEvent) {
+                        ItemStack fuel = furnaceBurnEvent.getFuel().clone();
+                        if (getTime() != EventValues.TIME_FUTURE)
+                            yield fuel;
+                        Material newMaterial = fuel.getType() == Material.LAVA_BUCKET ? Material.BUCKET : Material.AIR;
+                        fuel.setAmount(fuel.getAmount() - 1);
+                        if (fuel.getAmount() == 0)
+                            fuel = new ItemStack(newMaterial);
+                        yield fuel;
+                    }
+                    yield super.getItem();
+                    // a single lava bucket becomes an empty bucket
+                    // see https://minecraft.wiki/w/Smelting#Fuel
+                    // this is declared here because setting the amount to 0 may cause the ItemStack to become AIR
+                }
+                case RESULT -> {
+                    if (event instanceof FurnaceSmeltEvent furnaceSmeltEvent) {
+                        ItemStack result = furnaceSmeltEvent.getResult().clone();
+						ItemStack currentResult = ((FurnaceInventory) getInventory()).getResult();
```

Comment:
crazy indentation or just github?

Reply:
probably crazy indentation, IntelliJ has a habit of when pasting to set the indentation as spaces